### ISSUE TITLE
Syntax highlighting bug with single and double quotes

### ISSUE THREAD

**Issue by brian-carroll** [Krzysztof-Cieslak/vscode-elm#244]
Hey, great extension, thanks for creating it!
I just installed v0.20.0 and I noticed a syntax highlighting issue where I've got a `Char` literal for the double-quote character. So it's single quotes wrapped around a double quote, which I guess is a tricky input for a parser! I don't remember seeing this before in the same code though, so I wonder if your previous versions handled it?

![elm-vscode-highlighting-bug](https://user-images.githubusercontent.com/4647158/44616154-89529a00-a842-11e8-858c-aa23a132b301.PNG)

**Comment by 7sharp9** [comment]
Nothing has changed in the highlight cide for 20, there washould a
highlight fix some versions ago.  You could try and earlier version to see.
..

**Comment by brian-carroll** [comment]
Yes this looks like a bug in the grammar rather than actual highlighting. I guess there is some rule that says a double quote _always_ indicates the start of a string. But in this case, it's not the start of a string because it's the actual _character_ `'"'`.

**Comment by justgook** [comment]
Btw same problem for triple quotes ` “””” some “quoted” text “”””`
